<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClass</class>
 <widget class="QDialog" name="DialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="radiusLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radiusLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="areaLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="countBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dialog.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_countBtn_clicked()</slot>
 </slots>
</ui>
